Author: Chris Kuehl <ckuehl@ocf.berkeley.edu>
Subject: Remove Python 2 dependencies
Forwarded: not-needed
Description: Remove upstream declared dependencies which only apply to Python
 2.6. Since we build exclusively for Python 2.7 and Python 3, we don't need
 these.

--- a/setup.py
+++ b/setup.py
@@ -23,7 +23,6 @@
     packages=['aspy', 'aspy.yaml'],
     namespace_packages=['aspy'],
     install_requires=[
-        'ordereddict',
         'pyyaml',
     ],
 )
